exercises/01-composite-ui/before/Divergent.Frontend/app/modules/sales/orders/ordersController.js

```diff
@@ -6,13 +6,19 @@
 
                 var ctrl = this;
 
-                ctrl.isBusy = null;
+                ctrl.loadingData = true;
                 ctrl.orders = null;
 
-                ctrl.isBusy = backendCompositionService
+                backendCompositionService
                     .get('orders-list', { pageIndex: 0, pageSize: 10 })
                     .then(function (viewModel) {
                         ctrl.orders = viewModel.orders;
+                    })
+                    .catch(function (error) {
+                        $log.error('Ooops, something went wroing: ', error);
```

Comment:
I agree that we should do _something_ in the UI in this case. I'm not sure exactly what's involved in your "bindable" suggestion, but it _sounds_ good. 😉